src/content/en/updates/2017/02/lighthouse-jan.md

```diff
@@ -0,0 +1,147 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: What's new in Lighthouse 1.5.
+
+{# wf_updated_on: 2017-02-10 #}
+{# wf_published_on: 2017-02-10 #}
+{# wf_tags: lighthouse,tools,performance,progressive-web-apps #}
+{# wf_featured_image: /web/progressive-web-apps/images/pwa-lighthouse.png #}
+{# wf_featured_snippet: What's new in Lighthouse 1.5. New audits, extension updates, Performance Experiment, online Viewer features, and UI tweaks. #}
+
+# Lighthouse January 2017 update {: .page-title }
+
+{% include "web/_shared/contributors/brendankenny.html" %}
+{% include "web/_shared/contributors/ericbidelman.html" %}
+
+[Lighthouse](/web/tools/lighthouse/) is an
+[open-source](https://github.com/GoogleChrome/lighthouse), automated tool for
+improving the quality of your web apps. You can install it as a
+[Chrome Extension][crx] or run it as a Node command line tool. When you
+give Lighthouse a URL, it runs a barrage of tests against the page and then
+generates a report explaining how well the page did and indicating areas for
+improvement.
+{: .note }
+
+<img src="/web/progressive-web-apps/images/pwa-lighthouse.png"
+     class="lighthouse-logo attempt-right" alt="Lighthouse Logo">
+
+<style>
+.lighthouse-logo {
+  height: 150px;
+  width: auto;
+}
+</style>
+
+Today, we're happy to announce the [1.5 release](https://github.com/GoogleChrome/lighthouse/releases/tag/1.5.0) of Lighthouse, a huge release, with over **128 PRs**. Lighthouse 1.5 includes a bunch of big new features, audits, and the usual bug fixes. You can install it from npm (`npm i -g lighthouse`) or [download the extension](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk) from the Chrome Web Store.
```

Comment:
I do it in Sublime. I set up a key command for the menu item. I still have to do a tiny bit of fussing around links since the need to be on one line. This works 99% of the time. Go to Sublime Text > Preferences > Key Bindings - User and add: 

```
[
	{ "keys": ["ctrl+w"], "command": "wrap_lines", "args": {"width": 80}}
]
```